frontend/src/components/common/carousel/styles.scss

```diff
@@ -0,0 +1,102 @@
+$swiper-width: 540px;
+$font-lato: lato, arial, sans-serif;
+$z-index-current-image: 1;
+$preview-image-height: 70px;
+$main-image-height: 380px;
```

Comment:
Please use native CSS variables. Because we already use them to define root constants. We shouldn't use both the SCSS and CSS variables at the same time